src/title-nav/block.spec.js

```diff
@@ -0,0 +1,144 @@
+// title-nav test
+jest.setTimeout(30000)
+import {
+    createNewPost,
+    enablePageDialogAccept,
+    getEditedPostContent,
+    insertBlock,
+} from '@wordpress/e2e-test-utils';
+
+import {
+    clickElementByText,
+    blockStartup,
+    updateRangeInput,
+    clickCheckbox,
+    openSidebarPanelWithTitle
+} from '../test-helpers'
+
+const block = {blockTitle: 'Titled Navigation', blockName: 'purdue-blocks/title-nav'}
+
+describe( 'Titled Navigation Block', () => {
+    beforeAll( async () => {
+        await enablePageDialogAccept();
+    } );
+    beforeEach( async () => {
+        await createNewPost();
+    } );
+
+    it( 'Block should be available.', async () => {
```

Comment:
I think these it blocks could be a little more readable to the user in order to make the test more valuable. 

for example

```
describe('Google', () => {
  beforeAll(async () => {
    await page.goto('https://google.com');
  });

  it('should be titled "Google"', async () => {
    await expect(page.title()).resolves.toMatch('Google');
  });
});
```
https://jestjs.io/docs/en/puppeteer